notebooks/pluto/Project.toml

```diff
@@ -0,0 +1,6 @@
+[deps]
+GeoStats = "dcc97b0b-8ce5-5539-9008-bb190f959ef6"
+GeoStatsImages = "7cd16168-b42c-5e7d-a585-4f59d326662d"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
+Pluto = "c3e4b0f8-55cb-11ea-2926-15256bba5781"
```

Comment:
I think we don't need Pluto as a dependency, just PlutoUI, can you double check?

Reply:
I thought it will be more intuitive to add Pluto too,
as it will be smooth for the User to directly launch the Pluto from the same terminal (right in the same environment)
and he doesn't need to do `] add Pluto` again

Reply:
What I am trying to say is that Pluto isn't a dependency of the tutorials, and people willing to run these notebooks will have Pluto installed globally as opposed to locally here. Appreciate if you can remove the dependency.

Reply:
Sure, I will remove the dependency. 👍